### ISSUE TITLE
MissingPluginException when resuming the app from the background on Android

### ISSUE THREAD

**Issue by Brandon-Wolff** [Diversido/flutter_twilio_conversations#11]
This exception is thrown for every subscribed channel when resuming the app: 

```
I/flutter (13567): The following MissingPluginException was thrown while activating platform stream on channel
I/flutter (13567): flutter_twilio_conversations/CHXXXXXXXXXXXXXXXXXXXXXXX:
I/flutter (13567): MissingPluginException(No implementation found for method listen on channel
I/flutter (13567): flutter_twilio_conversations/CHXXXXXXXXXXXXXXXXXXXXXXX)
I/flutter (13567):
I/flutter (13567): When the exception was thrown, this was the stack:
I/flutter (13567): #0      MethodChannel._invokeMethod
platform_channel.dart:308
I/flutter (13567): <asynchronous suspension>
I/flutter (13567): #1      EventChannel.receiveBroadcastStream.<anonymous closure>
platform_channel.dart:652
I/flutter (13567): <asynchronous suspension>
```
After that, I have to completely close and relaunch the app to synchronize the channels.
Clearing both `channelChannels` and `channelListeners` in `onDetachedFromEngine` fixes this issue.

**Comment by Brandon-Wolff** [comment]
I made a PR for this fix: #12 
Is there a rationale for leaving those two HashMaps when the app is detached?

**Comment by asmodeoux** [comment]
Merged & new version of package is uploaded to pub.dev, thanks! ✅
